src/standalone/Snow/Snow.jl

```diff
@@ -203,12 +259,12 @@ such that SWE cannot become negative and U cannot become unphysical. The
 clipped values are what are actually applied as boundary fluxes, and are stored in
 `applied_` fluxes.
 """
-auxiliary_vars(::SnowModel) = (
+auxiliary_vars(m::SnowModel) = (
```

Comment:
this is nitpicky (sorry!), but we can just to ::SnowModel since `m` is not needed in the function. it will just dispatch off of type